src/main/java/nl/dtls/fairdatapoint/api/config/RestApiContext.java

```diff
@@ -105,18 +130,18 @@ public void configureContentNegotiation(ContentNegotiationConfigurer configurer)
             converter.configureContentNegotiation(configurer);
         }
     }
-    
-    @Bean( destroyMethod = "shutdownNow")
+
+    @Bean(destroyMethod = "shutdownNow")
     public Executor threadPoolTaskExecutor(@Value("${threadPoolSize:4}") int threadPoolSize) {
         return Executors.newFixedThreadPool(threadPoolSize);
     }
 
     @Bean(name = "publisher")
     public Agent publisher(@Value("${metadataProperties.publisherURI:nil}") String publisherURI,
             @Value("${metadataProperties.publisherName:nil}") String publishername) {
+        
         Agent publisher = null;
-        if (!publisherURI.contentEquals("nil")
-                && !publishername.contentEquals("nil")) {
+        if (!publisherURI.contentEquals("nil") && !publishername.contentEquals("nil")) {

```

Comment:
The `"nil"` literal is repeated a couple of times, it can be refactored into a constant.